src/js/views/draftScouting.js

```diff
@@ -26,7 +26,10 @@ async function addSeason(season, tid) {
         const pa = playersAll[i];
 
         // Abbreviate first name to prevent overflows
-        pa.name = `${pa.firstName.split(" ").map(s => s[0]).join(".")}. ${pa.lastName}`;
+        const names = pa.firstName.split(/[ .-]/);
+        const abbrev = names.map(s => {return s === "" ? "" : `${s[0]}.`;});
+        const concat = abbrev.join("");
+        pa.name = `${concat} ${pa.lastName}`;
```

Comment:
It would probably be good to break this into its own function so it was more clear what it was doing in isolation.